adoc/admin-security-role-management.adoc

```diff
@@ -226,3 +226,17 @@ roleRef:
 
 
 ====
+
+[IMPORTANT]
+====
+When creating new Roles, ClusterRoles, and bindings, it is important to keep in
```

Comment:
Should "Binding" also be capitalized?

Reply:
It would have to be RoleBindings and ClusterRoleBindings, which starts to get long but maybe is clearer